index.html

```diff
@@ -490,7 +502,8 @@ <h2><dfn>MediaDevices</dfn> Additions</h2>
                 <li>
                   <p>This invocation of {{MediaDevices/getDisplayMedia()}} is
                   now considered to have produced a new
-                  <dfn>capture-session</dfn>.</p>
+                  <dfn>capture-session</dfn>, with the associated
+                  [=capture-source-set=], <var>sources</var>.</p>
                 </li>
```

Comment:
Should the part below, which dealt with `controller.[[Source]]`, be updated to reference the video source from `capture-source-set` rather than (as it does now) the "video track's source"?
(GitHub didn't let me attach this comment in the desired location.)